### PR TITLE
Mid-September pull

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Added 12 lists and 1 syntax.
* Renamed 5 syntaxes.
* Removed 3 syntaxes.
* Replaced the two SVGs from the previous pull with more optimised file contents.
* Other minor amendments.

### CODE REVIEW

**Comment by collinbarrett** [comment]
there's another issue with the Directory Migrate check. actively fixing it. just a moment...

File: services/Directory/data/Syntax.json [hunk 1]
```diff
@@ -134,13 +134,9 @@
     "id": 31,
     "name": "URLRedirector"
   },
-  {
-    "id": 32,
```
Comment (collinbarrett) [inline services/Directory/data/Syntax.json:138]: I _think_ the build error is now something tied to removing this syntax. Maybe there's another link still to `32` that also needs to be removed.

**Comment by DandelionSprout** [comment]
Okay, so it just struck me that I had forgot to add view links for the new lists.

I must admit that having 6 different files for filterlist metadata has become a pretty big mess that even I am having a hard time learning and remembering.

**Comment by collinbarrett** [comment]
I totally agree. #372 is still probably the most important feature that needs to get done, and nobody has felt that pain more than you, I'm sure. it's just a big task that I've had a hard time carving out the time to tackle. still plan to when I can.

**Pull request merged**